styles/pinterest/catppuccin.user.css

```diff
@@ -164,6 +160,9 @@
     /* the void (parts not loaded yet) */
     html {
```

Comment:
Instead, change the selectors above to use `:root` like you have been, and replace the `html` here (and any other `html`s in this file) with `&`.